### PR TITLE
[ADD] clinic_management: add new module

### PR DESCRIPTION
**Pull request by francoleyes**

### CODE REVIEW

File: clinic_management/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    'name': 'Clinic Management',
+    'version': "17.0.1.0.0",
+    'author': 'Franco Leyes - Augusto cáceres - Santiago Agüero',
```
Comment (AugustoCacerees) [inline clinic_management/__manifest__.py]: Augusto Caceres

File: clinic_management/static/src/components/video_widget/video_widget.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    get url() {
+        return isBinarySize(this.value)
+            ? url('/web/content', {
+                model: this.props.record.resModel,
+                id: this.props.record.resId,
+                field: this.props.name,
+            })
+            : URL.createObjectURL(base64ToBlob(this.value, 'video/mp4'));
```
Comment (AugustoCacerees) [inline clinic_management/static/src/components/video_widget/video_widget.js]: ¿Qué ocurre si esta conversión tira error?

Comment (francoleyes) [reply clinic_management/static/src/components/video_widget/video_widget.js]: Ahí le agregué un chequeo con try catch

**Comment by out-of-reality** [comment]
[Pull request status dashboard](http://runbot.francoleyes.xyz/out-of-reality/out-of-reality/pull/7)

**Comment by out-of-reality** [comment]
@francoleyes I didn't know about this PR and had to retrieve its information, you may have to re-approve it as I didn't see previous commands.

**Comment by francoleyes** [comment]
@roboreality r+

**Pull request closed without merging**